{"level":"error","message":"apollo.model.validator.invalidvalue: Invalid Value: \"9\" for Field: eventType (Type: text) apollo.model.validator.invalidvalue: Invalid Value: \"9\" for Field: eventType (Type: text)","stack":"Error: apollo.model.validator.invalidvalue: Invalid Value: \"9\" for Field: eventType (Type: text)\n    at C:\\repo\\r\\controllers\\event.controller.js:40:19\n    at Object.f [as callback_or_throw] (C:\\repo\\r\\node_modules\\express-cassandra\\lib\\utils\\parser.js:25:5)\n    at C:\\repo\\r\\node_modules\\express-cassandra\\lib\\utils\\parser.js:290:14\n    at Array.some (<anonymous>)\n    at Object.fn [as get_save_value_expression] (C:\\repo\\r\\node_modules\\express-cassandra\\lib\\utils\\parser.js:253:50)\n    at f.fn [as save] (C:\\repo\\r\\node_modules\\express-cassandra\\lib\\orm\\base_model.js:1059:38)\n    at post (C:\\repo\\r\\controllers\\event.controller.js:38:11)\n    at newFn (C:\\repo\\r\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\repo\\r\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\repo\\r\\node_modules\\express\\lib\\router\\route.js:144:13)\n    at Route.dispatch (C:\\repo\\r\\node_modules\\express\\lib\\router\\route.js:114:3)\n    at newFn (C:\\repo\\r\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (C:\\repo\\r\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at C:\\repo\\r\\node_modules\\express\\lib\\router\\index.js:284:15\n    at Function.process_params (C:\\repo\\r\\node_modules\\express\\lib\\router\\index.js:346:12)\n    at next (C:\\repo\\r\\node_modules\\express\\lib\\router\\index.js:280:10)"}
